keyboards/40percentclub/luddite/keymaps/tzarc/keymap.c

```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
I think this can be safely removed
```suggestion
```

Reply:
Removed, applied same fix to default keymap, fixed formatting in the process.